Description: add missing whatis entries
 and fix other POD issued, seen in Perl 5.18
Origin: vendor
Author: gregor herrmann <gregoa@debian.org>
Reviewed-by: Jonathan Yu <jawnsy@cpan.org>
Last-Update: 2013-06-04

--- a/lib/PDF/API2/Annotation.pm
+++ b/lib/PDF/API2/Annotation.pm
@@ -13,7 +13,7 @@
 
 =head1 NAME
 
-PDF::API2::Annotation
+PDF::API2::Annotation - annotation management
 
 =head1 METHODS
 
--- a/lib/PDF/API2/NamedDestination.pm
+++ b/lib/PDF/API2/NamedDestination.pm
@@ -13,7 +13,7 @@
 
 =head1 NAME
 
-PDF::API2::NamedDestination
+PDF::API2::NamedDestination - PDF::API2 named destination support
 
 =head1 METHODS
 
--- a/lib/PDF/API2/Outline.pm
+++ b/lib/PDF/API2/Outline.pm
@@ -11,7 +11,7 @@
 
 =head1 NAME
 
-PDF::API2::Outline
+PDF::API2::Outline - outline management
 
 =head1 METHODS
 
--- a/lib/PDF/API2/Page.pm
+++ b/lib/PDF/API2/Page.pm
@@ -17,7 +17,7 @@
 
 =head1 NAME
 
-PDF::API2::Page
+PDF::API2::Page - page management
 
 =head1 METHODS
 
--- a/lib/PDF/API2/Resource.pm
+++ b/lib/PDF/API2/Resource.pm
@@ -11,7 +11,7 @@
 
 =head1 NAME
 
-PDF::API2::Resource
+PDF::API2::Resource - generic PDF::API2 resource class
 
 =head1 METHODS
 
--- a/lib/PDF/API2/Resource/BaseFont.pm
+++ b/lib/PDF/API2/Resource/BaseFont.pm
@@ -13,7 +13,7 @@
 
 =head1 NAME
 
-PDF::API2::Resource::BaseFont
+PDF::API2::Resource::BaseFont - base font class
 
 =head1 METHODS
 
--- a/lib/PDF/API2/Resource/CIDFont.pm
+++ b/lib/PDF/API2/Resource/CIDFont.pm
@@ -13,7 +13,7 @@
 
 =head1 NAME
 
-PDF::API2::Resource::CIDFont
+PDF::API2::Resource::CIDFont - base class for CID fonts
 
 =head1 METHODS
 
--- a/lib/PDF/API2/Resource/CIDFont/CJKFont.pm
+++ b/lib/PDF/API2/Resource/CIDFont/CJKFont.pm
@@ -16,7 +16,7 @@
 
 =head1 NAME
 
-PDF::API2::Resource::CIDFont::CJKFont
+PDF::API2::Resource::CIDFont::CJKFont -  - base class for CJK fonts
 
 =head1 METHODS
 
--- a/lib/PDF/API2/Resource/CIDFont/TrueType.pm
+++ b/lib/PDF/API2/Resource/CIDFont/TrueType.pm
@@ -12,7 +12,7 @@
 
 =head1 NAME
 
-PDF::API2::Resource::CIDFont::TrueType
+PDF::API2::Resource::CIDFont::TrueType - TrueType font support for PDF::API2
 
 =head1 METHODS
 
--- a/lib/PDF/API2/Resource/ColorSpace.pm
+++ b/lib/PDF/API2/Resource/ColorSpace.pm
@@ -11,7 +11,7 @@
 
 =head1 NAME
 
-PDF::API2::Resource::ColorSpace
+PDF::API2::Resource::ColorSpace - base color space support class for PDF::API2
 
 =head1 METHODS
 
--- a/lib/PDF/API2/Resource/ColorSpace/Indexed/ACTFile.pm
+++ b/lib/PDF/API2/Resource/ColorSpace/Indexed/ACTFile.pm
@@ -9,9 +9,11 @@
 
 no warnings qw[ deprecated recursion uninitialized ];
 
+=encoding latin1
+
 =head1 NAME
 
-PDF::API2::Resource::ColorSpace::Indexed::ACTFile
+PDF::API2::Resource::ColorSpace::Indexed::ACTFile - adobe color table support for PDF::API2
 
 =head1 METHODS
 
@@ -26,6 +28,8 @@
 November 2000
 for details.
 
+=back
+
 =cut
 
 sub new {
--- a/lib/PDF/API2/Resource/ColorSpace/Separation.pm
+++ b/lib/PDF/API2/Resource/ColorSpace/Separation.pm
@@ -11,7 +11,7 @@
 
 =head1 NAME
 
-PDF::API2::Resource::ColorSpace::Separation
+PDF::API2::Resource::ColorSpace::Separation - separation colorspace support for PDF::API2
 
 =head1 METHODS
 
--- a/lib/PDF/API2/Resource/ExtGState.pm
+++ b/lib/PDF/API2/Resource/ExtGState.pm
@@ -11,7 +11,7 @@
 
 =head1 NAME
 
-PDF::API2::Resource::ExtGState
+PDF::API2::Resource::ExtGState - extgate support for PDF::API2
 
 =head1 METHODS
 
--- a/lib/PDF/API2/Resource/Font.pm
+++ b/lib/PDF/API2/Resource/Font.pm
@@ -13,7 +13,7 @@
 
 =head1 NAME
 
-PDF::API2::Resource::Font
+PDF::API2::Resource::Font - Font support for PDF::API2
 
 =head1 METHODS
 
--- a/lib/PDF/API2/Resource/UniFont.pm
+++ b/lib/PDF/API2/Resource/UniFont.pm
@@ -8,7 +8,7 @@
 
 =head1 NAME
 
-PDF::API2::Resource::UniFont
+PDF::API2::Resource::UniFont - UniFont support class for PDF::API2
 
 =head1 METHODS
 
--- a/lib/PDF/API2/Resource/XObject.pm
+++ b/lib/PDF/API2/Resource/XObject.pm
@@ -11,7 +11,7 @@
 
 =head1 NAME
 
-PDF::API2::Resource::XObject
+PDF::API2::Resource::XObject - XObject support for PDF::API2
 
 =head1 METHODS
 
--- a/lib/PDF/API2/Resource/XObject/Form.pm
+++ b/lib/PDF/API2/Resource/XObject/Form.pm
@@ -11,7 +11,7 @@
 
 =head1 NAME
 
-PDF::API2::Resource::XObject::Form
+PDF::API2::Resource::XObject::Form - form support for PDF::API2
 
 =head1 METHODS
 
--- a/lib/PDF/API2/Resource/XObject/Form/BarCode.pm
+++ b/lib/PDF/API2/Resource/XObject/Form/BarCode.pm
@@ -11,7 +11,7 @@
 
 =head1 NAME
 
-PDF::API2::Resource::XObject::Form::BarCode
+PDF::API2::Resource::XObject::Form::BarCode - generic barcode support for PDF::API2 forms
 
 =head1 METHODS
 
--- a/lib/PDF/API2/Resource/XObject/Form/BarCode/code128.pm
+++ b/lib/PDF/API2/Resource/XObject/Form/BarCode/code128.pm
@@ -8,7 +8,7 @@
 
 =head1 NAME
 
-PDF::API2::Resource::XObject::Form::BarCode::code128
+PDF::API2::Resource::XObject::Form::BarCode::code128 - code128 barcode support for PDF::API2
 
 =head1 METHODS
 
--- a/lib/PDF/API2/Resource/XObject/Image.pm
+++ b/lib/PDF/API2/Resource/XObject/Image.pm
@@ -11,7 +11,7 @@
 
 =head1 NAME
 
-PDF::API2::Resource::XObject::Image
+PDF::API2::Resource::XObject::Image - image support for PDF::API2
 
 =head1 METHODS
 
--- a/lib/PDF/API2/Resource/XObject/Image/TIFF.pm
+++ b/lib/PDF/API2/Resource/XObject/Image/TIFF.pm
@@ -13,7 +13,7 @@
 
 =head1 NAME
 
-PDF::API2::Resource::XObject::Image::TIFF
+PDF::API2::Resource::XObject::Image::TIFF - tiff support for PDF::API2
 
 =head1 METHODS
 
